GOVERNANCE.md

```diff
@@ -1,98 +1,171 @@
+Any community member or contributor can ask that something be reviewed
+by the CC by logging a GitHub issue. Any Collaborator, CC member, or the
+meeting chair can bring the issue to the CC's attention by applying the
+`cc-review` label. If consensus-seeking among CC members fails for a
+particular issue, it may be added to the CC meeting agenda by adding the
+`cc-agenda` label.
 
-The CC will elect from amongst voting CC members a CC Chairperson to work on building an agenda for CC meetings and collaborate with the Individual Membership Directors the wishes of the CC to the Board for a term of one year according to the Node.js Foundation’s By-laws. The CC shall hold annual elections to select a CC Chairperson; there are no limits on the number of terms a CC Chairperson may serve.
+Prior to each CC meeting, the meeting chair will share the agenda with
+members of the CC. CC members can also add items to the agenda at the
+beginning of each meeting. The meeting chair and the CC cannot veto or remove
+items.
 
-## Section 8. Voting
+The CC may invite additional persons to participate in a non-voting capacity.
 
-For internal project decisions, Collaborators shall operate under Lazy Consensus. The CC shall establish appropriate guidelines for implementing Lazy Consensus (e.g. expected notification and review time periods) within the development process.
+The meeting chair is responsible for ensuring that minutes are taken and that a
+pull request with the minutes is submitted after the meeting.
 
-The CC follows a Consensus Seeking decision making model. When an agenda item has appeared to reach a consensus the moderator will ask "Does anyone object?" as a final call for dissent from the consensus.
+Due to the challenges of scheduling a global meeting with participants in
+several timezones, the CC will seek to resolve as many agenda items as possible
+outside of meetings using
+[the Community-Committee issue tracker](https://github.com/nodejs/community-committee/issues). The process in
+the issue tracker is:
 
-If an agenda item cannot reach a consensus a CC member can call for either a closing vote or a vote to table the issue to the next meeting. The call for a vote must be seconded by a majority of the CC or else the discussion will continue. Simple majority wins.
+* A CC member opens an issue explaining the proposal/issue and @-mentions
+  @nodejs/community-committee.
+* After 72 hours, if there are two or more `LGTM`s from other CC members and no
+  explicit opposition from other TSC members, then the proposal is approved.
```

Comment:
I'd like to object to this line explicitly. `TSC` should be `CommComm`.

Reply:
Yeah that was a copy paste error